### ISSUE TITLE
Accessible links inside the carousel

### ISSUE THREAD

**Issue by camipe** [baianat/hooper#128]
**Is your feature request related to a problem? Please describe.**
I'm using Hooper to display a set of thumbnails, when the user clicks on one a video player is opened. My problem is to fullfill WCAG guideline 2.1 the application needs to be accessible using a keyboard only but when using Hooper I can't "click" on the links using enter. 

**Describe the solution you'd like**
It should be possible to navigate the slides using the keyboard and pressing enter to open the link inside a slide.

**Additional context**
I'm not sure if this is possible already? I've tried a bunch of different settings from the documentation but none seems to do the trick.
